pkg/spec/createconfig.go

```diff
@@ -405,7 +405,9 @@ func (c *CreateConfig) GetContainerCreateOptions() ([]libpod.CtrCreateOption, er
 	// does not have one
 	options = append(options, libpod.WithEntrypoint(c.Entrypoint))
 
-	if c.NetMode.IsContainer() {
+	if os.Getuid() != 0 {
```

Comment:
It would be nice if these guards were based on [capabilities][1] instead of UIDs.  In this case, I'm guessing that's `CAP_NET_BIND_SERVICE` and/or `CAP_NET_ADMIN` in the host network namespace.

[1]: http://man7.org/linux/man-pages/man7/capabilities.7.html

Reply:
This can't be our leading `if` clause - it will match always as long as we are not rootless, when we don't want to include a network ns if we are `--net=host`, `--net=container`, `--net=none`